File: FinalProject/ExchangeBook/Data/Person.cs
namespace ExchangeBook.Data
{
    public class Person
    {
        public int Id { get; set; }
        public string? Firstname { get; set; }
        public string? Lastname { get; set; }
        public string? PhoneNumber {  get; set; }

        public int? UserId { get; set; }
        public virtual User? User { get; set; }

        public virtual ICollection<Book> Books { get; } = new HashSet<Book>();

        public override string? ToString()
        {
            return $"{Firstname}, {Lastname}, {PhoneNumber}";
        }
    }
}

File: angular/book-Exchange/src/app/shared/interfaces/person.ts
import { User } from "./user";

export interface Person {
  id:number
  firstname: string;
  lastname: string;
  phoneNumber: string;
  userId:number;


}
export interface UserSignUpDTO{
  Username:string;
  Email:string
  Password:string;
  Firstname:string;
  Lastname:string;
  PhoneNumber:string;
  UserRole:string;
}